addons/l10n_ae/data/account.account.template.csv

```diff
@@ -1,183 +1,158 @@
-"uae_account_3911","Loss on Difference on Exchange","531102","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3912","Disposal of Business Branch","531103","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3914","Royalty to Parent Co.","531201","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3916","Income Tax","531301","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3917","Zakat","531302","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3919","Previous Year Adjustments Account","531411","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3921","Other Non Operating Expenses","531511","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3924","Credit Card Charges","541001","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3925","Amex Credit Cards Charges","541002","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3926","Bank Finance & Loan Charges","541003","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3927","Air Miles Card Charges","541004","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3928","Credit Card Swipe Charges","541005","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3929","PayPal Charges","541006","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3932","Amortization on Leasehold Improvement","551001","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3933","Depreciation Of Furniture & Office Equipment","551002","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3934","Depreciation Of Computer Hard & Soft","551003","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3935","Depreciation Of Motor Vehicles","551004","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3937","Consultancy Fees","560001","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3938","Provision for Doubtful Debts","560004","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
-"uae_account_3941","Closing Account","611001","account.data_account_type_expenses","l10n_ae.uae_chart_template_standard","False"
+id,name,code,user_type_id/id,chart_template_id/id,reconcile
+uae_account_3665,Right of use Asset (IFRS 16),100101,account.data_account_type_fixed_assets,l10n_ae.uae_chart_template_standard,False
```

Comment:
Why is the `id` the same but the `code` changes? It is likely to break links elsewhere because we don't notice that we are not referencing the same account anymore.

Reply:
Do you think it would be a better idea to make the `id` match the `code`?

Reply:
It would be a good idea indeed; I have no clue what the previous ids were referring to.

However, you can't remove records in a stable version. What you can do is keep them, but set the field `nocreate` to `True` on them.